src/smt/smt_engine.cpp

```diff
@@ -1131,6 +1131,16 @@ void SmtEngine::setDefaults() {
         options::cbqi.set(true);
       }
     }
+    // setting unif requirements
+    if (options::sygusUnifBooleanHeuristicDt()
```

Comment:
Can you merge master and then move this (and the above options setting in this block) to https://github.com/CVC4/CVC4/blob/master/src/smt/smt_engine.cpp#L1809 ?